scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -74,6 +74,7 @@ SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASK1KHZ : 3|1@1+ (1,0) [0|1] "" LOGGER
 SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANRX : 4|1@1+ (1,0) [0|1] "" LOGGER
 SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX : 5|1@1+ (1,0) [0|1] "" LOGGER
 SG_ WATCHDOG_TIMEOUT : 6|1@1+ (1,0) [0|1] "" LOGGER
+SG_ Wheel_Speed_Non_Critical_Fault : 7|1@1+ (1,0) [0|1] "" LOGGER
```

Comment:
I would make separates faults for left and right wheel speed

Reply:
Fixed